exprel/graph_extractor/extract.py

```diff
@@ -0,0 +1,331 @@
+import argparse
+import json
+import logging
+import sys
+import os
+from collections import defaultdict
+
+import networkx as nx
+import pandas as pd
+import stanza
+from exprel.dataset.utils import default_pn_to_graph, ud_to_graph
+from graphviz import Source
+from networkx.algorithms.isomorphism import DiGraphMatcher
+from sklearn.metrics import precision_recall_fscore_support
+from tqdm import tqdm
+from tuw_nlp.grammar.text_to_4lang import TextTo4lang
+from tuw_nlp.graph.utils import (
+    GraphFormulaMatcher,
+    GraphMatcher,
+    graph_to_isi,
+    graph_to_pn,
+    pn_to_graph,
+)
+from tuw_nlp.text.pipeline import CachedStanzaPipeline
+
+
+class GraphExtractor:
```

Comment:
the GraphExtractor class may also be a good candidate for tuw_nlp/graphs

Reply:
Yes it might be, but It will also contain amr parsing and I'm not sure if that would belong to the tuw-nlp repo. Your thoughts?